.travis.yml

```diff
@@ -1,25 +0,0 @@
-language: ruby
-
-before_install:
-  - gem i bundler
-
-bundler_args: --without benchmark --jobs=3 --retry=3
-
-matrix:
-  include:
-    - rvm: 2.4.9
-    - rvm: 2.5.7
-    - rvm: 2.6.5
-    - rvm: 2.7.0
-    - rvm: jruby-9.2.9.0
-      env: JRUBY_OPTS=--debug
-      jdk: openjdk8
-    - rvm: ruby-head
-    - rvm: jruby-head
-      env: JRUBY_OPTS=--debug
-      jdk: openjdk8
-  allow_failures:
-    - rvm: ruby-head
-    - rvm: jruby-head
```

Comment:
I haven't looked into github actions so far - is there a way for us to get back our checks against head? Truthfully because they were partially broken we didn't really use them so they're not a deal breaker but would be good to know

Reply:
Ouch, I forgot that! There's no currently "built-in way" to do that. So the options are:

* Keep TravisCI for ruby-head for the moment.
* Add a separate workflow that installs `rvm`, and ruby-head on top of it.
* Use the official nightly docker image: https://github.com/ruby/ruby-docker-images.

Reply:
Cool thanks, as I said not a deal breaker for me so I'm happy to go ahead without any of this but it's great to know that we got options :ok_hand:

Reply:
I migrated the old `ruby-head` and `jruby-head` old TravisCI entries to keep using RVM under github actions.